synapse/handlers/auth.py

```diff
@@ -455,11 +485,17 @@ def add_threepid(self, user_id, medium, address, validated_at):
     def _save_session(self, session):
         # TODO: Persistent storage
         logger.debug("Saving session %s", session)
+        session["last_used"] = time.time()
```

Comment:
Why not use `hs.get_clock()`?

We generally try and consistently use milliseconds internally, rather than seconds.